code/src/components/LikeButton.js

```diff
@@ -0,0 +1,30 @@
+import React, { useState } from 'react'
+import { Emoji } from './Emoji.js'
+
+export const LikeButton = ({ hearts, id, apiUrl }) => {
+
+  const [likes, setLikes] = useState(hearts)
+
+  const addLike = (event) => {
```

Comment:
In case you didn't know - you don't need the `()` when only passing one argument
```suggestion
  const addLike = event => {
```